script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -721,42 +726,43 @@ The extreme -100℃ conditions bring my entire body to a halt.
 [sha:b4e5150c5558ca7fb00de7ba18876e65b4be4794]{
 -- Page 701, Offset 10555.
 -- 着地に備えようとする本能も、
-My instinct to prepare for landing―――
+//Switched around a bit
+Both my instinct to prepare for landing,
+ and the rational thought to value my own life―――
 }
 [sha:461e0c66f605cafe1e758b72e6ab03cebb33feda]{
 -- Page 701, Offset 10556.
 -- 命の意義を唱える理性も、もう遙か上空に置いてきた。
-And the reasoning necessary to advocate for the significance of life
- have already been left behind with the night sky.
+―――have been left behind in the far above of the night sky.
 }
 [sha:3039bd95f611564ee3928fb9ab8c165cd8946fdb]{
 -- Page 702, Offset 10557.
 -- 「ヴローヴ……！」
-"Vlov...!"
+"Vlov!"
 }
 [sha:d77615b2ed4f6ac67a6a713cf6a4a9b43dbc9a11]{
 -- Page 702, Offset 10558.
 -- <腿|もも>から腰、腰から背筋、背筋からの運動エネルギーを、ナイフを持つ右手の振りに譲渡する。
 I transfer the kinetic energy from my thighs to my waist,
  from my waist to my back muscles,
- and from my back muscles to the swing of the knife in my right hand.
+ and finally, let my back muscles carry it all the way to the knife in my right hand.
 }
 [sha:a25b30e53e0a3ccaaaae29226180ac9e14bcb8aa]{
 -- Page 702, Offset 10559.
 -- もはや筋肉が停止しようと止まらない。
-My muscles no longer try to hold me back.
+//I considered 'My muscles could no longer stop me if they tried', which might be a bit closer to JP, but sounds bizarre.
+I could no longer stop my muscles if I tried.
```

Comment:
> I couldn't stop my muscles now[,] even if I tried.

or, if we want to go back towards the JP again,
> My muscles are no longer screeching at me to stop.

and change the next line to start with "I've become"

Reply:
The first read far less smoothly to me, and the second feels kinda wrong.

Reply:
Ehh this still reads weirdly to me. It comes off as past aspect (I was no longer able to stop my muscles).